src/niscope/metadata/enums_addon.py

```diff
@@ -52,91 +60,91 @@
                 },
             },
             {
-                'name': 'NISCOPE_VAL_480I_59_94_FIELDS_PER_SECOND',
+                'name': 'NISCOPE_VAL_VIDEO_480I_59_94_FIELDS_PER_SECOND',
```

Comment:
Are these here because they're missing from CVI file?

Reply:
No, not missing. Since we use both LV & CVI and they differ we get a superset of them. This is to limit to the portion we want.